content/_includes/server-config.md

```diff
@@ -366,4 +346,20 @@ keystore:
       # with Azure managed credentials.
       managed_identity:
         client_id: ""      # The Azure managed identity of the client - i.e. a UUID.
+
+  entrust:
+    # The Entrust KeyControl configuration.
+    # For more information take a look at:
+    # https://www.entrust.com/digital-security/key-management/keycontrol
+    keycontrol:
+      endpoint: ""     # The KeyControl endpoint - e.g. https://keycontrol.my-org.com
+      vault_id: ""     # The Vault ID            - e.g. e30497c1-bff7-4e81-beb7-fb35c4b7410c
+      box_id:   ""     # The Box name or ID      - e.g. tenant-1
+      # The KeyControl access credentials
+      credentials:
+        username: ""   # The username able to access the Vault and Box.
```

Comment:
maybe
```suggestion
        username: ""   # A username with access to the Vault and Box.
```

Also more `e.g.` in this area that could be replaced with words

Reply:
Replaced those and `i.e.` throughout. But will need to also address in the KES repo.